extension/src/components/Popup.tsx

```diff
@@ -3,41 +3,31 @@ import {
     AppBar,
     Card,
     CardContent,
-    CardHeader,
     CircularProgress,
     CssBaseline,
-    Link,
     Toolbar,
     Typography
 } from '@mui/material';
 import SettingsIcon from '@mui/icons-material/Settings';
-import wretch from 'wretch';
-import { PopupToBackgroundMsg, BackgroundToPopupMsg, MsgType, MLISResponse } from "../extension/types";
-
-// TODO: Issue #14: Change url and endpoint after MLIS is ready
-const apiRoot: string = "http://localhost:8000";
-const summarizationEndpoint: string = "/summary";
+import { MLISRequest, MLISResponse } from "../extension/types";
 
 function Popup() {
     const [isLoading, setIsLoading] = useState(true);
     const [content, setContent] = useState('');
     const [articleTitle, setArticleTitle] = useState('');
 
     useEffect(() => {
-        const msg: PopupToBackgroundMsg = {
-            type: MsgType.PopUpInit
-        };
-        chrome.runtime.sendMessage(msg, (response: BackgroundToPopupMsg) => {
-            if (response && response.textToSummarize) {
-                wretch(apiRoot + summarizationEndpoint)
-                    .options({ mode: "cors" })
-                    .post({ text: response.textToSummarize })
-                    .json((mlisResponse: MLISResponse) => {
-                        setArticleTitle(response.articleTitle);
-                        setContent(mlisResponse.text);
-                        setIsLoading(false);
-                    });
-            }
+        chrome.tabs.query({active: true, lastFocusedWindow: true}, tabs => {
+            const request: MLISRequest = {
+                url: tabs[0].url,
+            };
+            chrome.runtime.sendMessage(request, (response: MLISResponse) => {
+                if (response && response.articleSummary) {
```

Comment:
We should also consider a case when MLIS returns a response from [this if case](https://github.com/TeamCHK/yubaba/pull/23/files#diff-8da499e4bcea24668d71c7e2aae1badd86c32930f29c59799139bca1d94a537fL37)

Reply:
Now status code is added to `MLISResponse` in `background.ts`.
Then, we check the status code in Popup and treat the response differently.